### PR TITLE
Add options to selectively disable default scrubs

### PR DESCRIPTION
**Pull request by etceterum**

### CODE REVIEW

File: lib/logstop.rb [hunk 1]
```diff
@@ -14,19 +14,34 @@ module Logstop
...
+    options = DEFAULT_OPTIONS.merge(options)
 
     # order filters are applied is important
-    msg.gsub!(URL_PASSWORD_REGEX, FILTERED_URL_STR)
-    msg.gsub!(EMAIL_REGEX, FILTERED_STR)
-    msg.gsub!(CREDIT_CARD_REGEX, FILTERED_STR)
-    msg.gsub!(CREDIT_CARD_REGEX_DELIMITERS, FILTERED_STR)
-    msg.gsub!(PHONE_REGEX, FILTERED_STR)
-    msg.gsub!(SSN_REGEX, FILTERED_STR)
+    msg.gsub!(URL_PASSWORD_REGEX, FILTERED_URL_STR) if options[:password]
```
Comment (dknox20) [inline lib/logstop.rb:35]: Don't think there is a case to separate this out yet so the if's seem fine to me.

**Review by dknox20 — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -15,6 +15,15 @@ By default, scrubs:
 - Social Security numbers (SSNs)
 - passwords in URLs
 
+To disable any of the default scrubs above, use:
```
Comment (dknox20) [inline README.md:18]: Actually not sure should we fold in the IP option here?

File: lib/logstop.rb (hunk 1 above)
Comment (etceterum) [reply lib/logstop.rb:35]: I just wanted to make it as flexible as possible - I am looking from the perspective of not only addressing our current need, but to help the gem become more flexible overall

File: README.md (hunk 2 above)
Comment (etceterum) [reply README.md:18]: IP scrubbing is false by default; I wanted to preserve the conventions that already exist so, if our PR into the master repo is accepted and users upgrade to the new version, nothing breaks. If this was a new gem (or if we don't want our change merged into master) - I agree that it's better to treat all flags uniformly

**Pull request merged**